app/graphql/types/enum/mapping_external_site.rb

```diff
@@ -0,0 +1,17 @@
+class Types::Enum::MappingExternalSite < Types::Enum::Base
+  value 'MY_ANIME_LIST_ANIME', value: 'myanimelist/anime'
+  value 'MY_ANIME_LIST_MANGA', value: 'myanimelist/manga'
+  value 'ANILIST_ANIME', value: 'anilist/anime'
+  value 'ANILIST_MANGA', value: 'anilist/manga'
+  value 'THE_TVDB', value: 'thetvdb'
+  value 'THE_TVDB_SERIES', value: 'thetvdb/series'
+  value 'THE_TVDB_SEASON', value: 'thetvdb/season'
+  value 'ANI_DB', value: 'anidb'
+  value 'ANIME_NEWS_NETWORK', value: 'animenewsnetwork'
+  value 'MANGA_UPDATES', value: 'mangaupdates'
+  value 'HULU', 'hulu'
+  value 'IMDB_EPISODES', value: 'imdb/episodes'
+  value 'AOZORA', value: 'aozora'
+  value 'TRAKT', value: 'trakt'
+  value 'MY_DRAMA_LIST', value: 'mydramalist'
```

Comment:
Why are all these brand names being split apart? MyDramaList is _MyDramaList_ not _My Drama List_. Anime News Network is the only service that formats their brand with a space. 

Although I'd prefer to keep Anime News Network as `ANIMENEWSNETWORK` and the enum format be `siteName_specificResource` (all caps)

Reply:
I can keep them as one word, it just seemed weird when in all caps.

Reply:
It is rather weird but I think wopian has the right idea to use an underscore as the separator